needle/core/utils/constants.py

```diff
@@ -97,10 +97,26 @@ class Constants(object):
     # DEVICE
     # ==================================================================================================================
     # DEVICE PATHS
+<<<<<<< HEAD

```

Comment:
Hi, this is a conflict with the mainstream version. Please update your local copy to fix this issue